pikchr-mode.el

```diff
@@ -225,7 +225,10 @@ the block contents as BODY and its header argumenst as PARAMS."
       (insert body))
     (org-babel-eval
      (concat (org-babel-process-file-name pikchr-executable)
-             " --svg-only " (org-babel-process-file-name in-file))
+             " --svg-only "
+             (when (assq :dark-mode params) "--dark-mode ")
+             (when (assq :cmdline params) (cdr (assq :cmdline params)))
```

Comment:
I think we should add an extra space in case `:cmdline` is passed?
```suggestion
             (when-let ((cmdline (cdr (assq :cmdline params))))
               (concat cmdline " "))
```